src/Components/Login/LoginContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {getloginUserThunkCreator} from "../../redux/auth-reducer";
+import Login from "./Login";
+import {connect} from "react-redux";
+
+
+class LoginContainer extends React.Component {
+
+    componentDidMount() {
+        this.props.getloginUserThunkCreator();
+        //
+        // LoginAPI.postLogin()
+        //     .then(response => {
+        //             if (response.data.accessToken !== "") {
+        //                 let {accessToken, accessTokenExpiresIn, refreshToken, refreshTokenExpiresIn} = response.data;
+        //
+        //                 this.props.setAuthUserData(accessToken, accessTokenExpiresIn, refreshToken, refreshTokenExpiresIn)
+        //             }
+        //
+        //         }
+        //     );
+
+    }
+
+    render() {
+        return (
+            <Login/>
+        )
+    }
+
+}
+
+let mapStateToProps = (state) => ({
+    isAuth: state.auth.isAuth,
```

Comment:
You pass the result of this `mapStateToProps` and `mapDispatchToProps` to the `LoginContainer` component, but don't pass it from `LoginContainer` to the `Login`. Actually you render <Login/> without argument so `Login` will not get all this data from the store.